applications/GeoMechanicsApplication/custom_workflows/time_loop_executor.hpp

```diff
@@ -119,11 +117,11 @@ public :
         }
     }
 
-    std::unique_ptr<TimeIncrementor>  mTimeIncrementor;
-    std::function<bool()>             mCancelDelegate;
-    std::function<void(double)>       mProgressDelegate;
+    std::unique_ptr<TimeIncrementor> mTimeIncrementor;
+    std::function<bool()> mCancelDelegate;
+    std::function<void(double)> mProgressDelegate;
```

Comment:
It seems the old format is better. clang has 'bool Enabled'. Can we use it?

Reply:
Good suggestion, I missed the AlignConsecutiveDeclarations option when checking the documentation, so I didn't think it was possible!